packages/mui-joy/src/Typography/Typography.tsx

```diff
@@ -87,6 +87,7 @@ const TypographyRoot = styled('span', {
         }
       : {
           display: 'block', // don't rely on user agent, always `block`.
+          position: 'relative',
```

Comment:
Hey, I was playing around with the `<Link />` overlay prop. The demo is not working for me: https://mui.com/joy-ui/react-link/#overlay-prop

I figured out the parent element has to have `position: relative` in order to make the whole card clickable. As the `<Typography />` sets position relative, the overlay magic is not working anymore. 

I'm not 100% sure the problem is related to your change, but in any case, I wanted to leave the feedback here.

Reply:
Hey, thanks. I am fixing it!